v0/destinations/ga/transform.js

```diff
@@ -21,6 +21,8 @@ const {
 function getParamsFromConfig(message, destination, type) {
   const params = {};
   const obj = {};
+  const messageType = message.type;
+  const traits = message.traits || message.context.traits;
```

Comment:
use the util getFieldValueFromMessage ?
else we need to add a falsy check for message.context.

Reply:
done. Please check.